backend/api/job_board/types.py

```diff
@@ -0,0 +1,35 @@
+from datetime import datetime
+from typing import Optional
+
+import strawberry
+
+from ..helpers.i18n import make_localized_resolver
+
+
+@strawberry.type
+class JobListing:
+    id: strawberry.ID
+    title: str = strawberry.field(resolver=make_localized_resolver("title"))
+    slug: str = strawberry.field(resolver=make_localized_resolver("slug"))
+    description: str = strawberry.field(resolver=make_localized_resolver("description"))
+    company: str
+    company_logo: Optional[str]
+    apply_url: Optional[str]
```

Comment:
```suggestion
    apply_url: str
```

I don't think this should be null, if it isn't specified empty string